sass/_themes.scss

```diff
@@ -810,22 +814,26 @@ $themes: (
 
 :export {
     red: $color-red;
+    chart_baseline_up: $CHART_BASELINE_UP;
+    chart_baseline_down: $CHART_BASELINE_DOWN;
     light_chart_text: map-get(map-get($themes, 'light'), 'ChartText');
     dark_chart_text: map-get(map-get($themes, 'dark'), 'ChartText');
     light_chart_grid: map-get(map-get($themes, 'light'), 'ChartGrid');
     dark_chart_grid: map-get(map-get($themes, 'dark'), 'ChartGrid');
+    light_chart_default_border_closed: map-get(map-get($themes, 'light'), 'DefaultBorderClosed');
+    dark_chart_default_border_closed: map-get(map-get($themes, 'dark'), 'DefaultBorderClosed');
```

Comment:
Please check with Deveraj, beside of `Area` type the rest of the chart types seems to follow the same colors, green for up and red for down, as a result, we can have a more general definition for them.

Reply:
Also `DefaultBorder` is the color of the chart grid. better to use another naming for it if we suppose to add a new color for the chart types

Reply:
For the first part OK I will talk with him and share the result.
For the second part I agree with you.